flakey-test-tracking/src/config.js

```diff
@@ -0,0 +1,32 @@
+// shouldTrackFlakes => tests are retried `numRetries` times and flakey results are logged w/ test output
+const shouldTrackFlakes =
+  (process.env.CIRCLECI && process.env.FLAKEY !== 'false') || process.env.FLAKEY === 'true'
+
+// numRetries === times test is run after the initial failure
+const numRetries = process.env.NUM_RETRIES ? Number(process.env.NUM_RETRIES) : 25
+
+// shouldSkipKnownFlakes => flakey test issues are fetched from github and corresponding tests are skipped
+const shouldSkipKnownFlakes =
+  shouldTrackFlakes && process.env.CIRCLECI && process.env.SKIP_KNOWN_FLAKES !== 'false'
```

Comment:
Hmm okay yea I like that. I'll add it

Reply:
Actually, I think the way to do this would be to just set NUM_RETRIES=0 . Decoupling the <shouldTrackFlakes> logic and the shouldSkipTests logic would be non-trivial and I'm hesitant to make a change like that at this stage. What do you think about adding some documentation in the README explaining that if you want a 'fail fast without flaky' you should set NUM_RETRIES=0?

Reply:
Ok, if it's documented, should be fine.
Another way could be to add a new flag, and if that flag is set, you could assume the NUM_RETRIES=0. But both seems good to me, your choice.